clang/include/clang/Basic/LangOptions.h

```diff
@@ -303,6 +304,15 @@ class LangOptions : public LangOptionsBase {
 
   enum ExcessPrecisionKind { FPP_Standard, FPP_Fast, FPP_None };
 
+  enum FPAccuracyKind {
+    FPA_Default,
+    FPA_High,
+    FPA_Medium,
+    FPA_Low,
+    FPA_Sycl,
+    FPA_Cuda
```

Comment:
```suggestion
    FPA_Cuda,
```
as a kindness to the next person adding to this enumeration.